arkouda/dataframe.py

```diff
@@ -278,7 +278,13 @@ def __init__(self, initialdata=None, index=None):
             # convert the lists defining each column into a pdarray
             # pd.DataFrame.values is stored as rows, we need lists to be columns
             for key, val in initialdata.to_dict("list").items():
-                self.data[key] = array(val)
+                if isinstance(val[0], list):
+                    multi_array = []
+                    for r in val:
+                        multi_array.append(array(r))
```

Comment:
Stylistic, but would something like this work?
```python
multi_array = [array(r) for r in val]
```
i think we could also make this entire block one line if ppl like that style (black would break it up so it wouldn't be too long)
```python
self.data[key] = SegArray.from_multi_array([array(r) for r in val]) if isinstance(val[0], list) else array(val)
```